lib/matplotlib/lines.py

```diff
@@ -1242,26 +1241,6 @@ def set_dashes(self, seq):
         else:
             self.set_linestyle((0, seq))
 
-    def _draw_solid(self, renderer, gc, path, trans):
```

Comment:
I do not mind removing the methods, it the inner call to `gc.set_linestyle` that I am worried about.